packages/web-pkg/src/composables/actions/useBlockedPopupRedirectMessage.ts

```diff
@@ -0,0 +1,14 @@
+import { useGettext } from 'vue3-gettext'
+
+export const useBlockedPopupRedirectMessage = () => {
+  const { $gettext } = useGettext()
+
+  return {
+    title: $gettext('Blocked pop-ups and redirects'),
+    timeout: 20,
```

Comment:
@elizavetaRa 10s felt too short, so I doubled it. Still quite a lot of text to digest for the average user though